### DISCUSSION TITLE
ruan-cat-own-notes/vuepress/vuepress

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#116] [Announcements]
# ruan-cat-own-notes/vuepress/vuepress

功能细节优化 notes.youngkbt.cn/about/website/introduce; 参照这个链接内的教程配置。逐步地优化网站的交互效果。 想开发自己的插件实现标题序号自动生成 如图，期望菜单可以增加序号功能： 2023-11-21-17-01-57 比如： 为侧边栏提供序号。; 为每一个标题栏提供序号。即 h1,h2,h3,h4,h5,...

https://www.ruan-cat.com/ruan-cat-own-notes/vuepress/vuepress.html

**Comment by zeevenn** [comment]
一样的想法，我也想给文章自动加序号

**Comment by zeevenn** [comment]
我开发完了，发布到了 npm 上了，Vuepress Plugin Ordered Header
